README.md

```diff
@@ -217,13 +219,88 @@ The corresponding function in on Tarantool Cartridge router may look like (uses
 
 See more examples in the module tests.
 
+#### Call stored functions in Tarantool instance
+
+You can bind repository methods to calls of the stored functions in the Tarantool instance using the `@Query` annotation
+with the stored function name specified in the `functionName` parameter.
+For such methods, you can specify the stored function response format so that it will be parsed correctly. The response
+format may be either an object (and a list of objects) or a tuple (and a list of tuples).
+
+```java
+@Data
+@Tuple
+public class SampleUser {
+    private String name;
+    private String lastName;
+}
+```
+
+```java
+public interface SampleUserRepository extends TarantoolRepository<SampleUser, String> {
+    @Query(function = "returning_sample_user_object")
+    SampleUser returningSampleUserObject(String name);
+
+    @Query(function = "get_predefined_users")
+    List<SampleUser> getPredefinedUsers();
+}
+```
+
+```java
+@Tuple
+public class Book {
+  @Id
+  private Integer id;
+  private String name;
+}
+
+// by default, schema name is another_space for all CRUD standard methods
+@Tuple("another_space") 
+public interface BookRepository extends TarantoolRepository<Book, Integer> {
+    
+    // it overrides schema name to test_space for current method
+    @Tuple("test_space") 
+    @Query(function = "do_something")
+    Optional<Book> doSomething(Book book);
+}
+```
+
+By default, schema name is book_entity because schema name not specified 
```

Comment:
```suggestion
Here the space name is not specified neither in a `@Tuple` annotation on the repository interface, nr on the entity class, so it will be "book_entity" according to the entity class name, turned into "lower snake case".
```